../../third_party/perfetto/src/trace_processor/dataframe/specs.h:68:30: error: unknown type name 'nullptr_t'; did you mean 'std::nullptr_t'?
   68 |   using Value = std::variant<nullptr_t, int64_t, double, const char*>;
      |                              ^~~~~~~~~
      |                              std::nullptr_t
/usr/bin/../lib/gcc/x86_64-linux-gnu/14/../../../../include/x86_64-linux-gnu/c++/14/bits/c++config.h:314:29: note: 'std::nullptr_t' declared here
  314 |   typedef decltype(nullptr)     nullptr_t;
      |                                 ^
1 error generated.

--- a/third_party/perfetto/src/trace_processor/dataframe/specs.h
+++ b/third_party/perfetto/src/trace_processor/dataframe/specs.h
@@ -65,7 +65,7 @@ using Nullability = TypeSet<NonNull>;
 // This is used to generate query plans for filtering rows.
 struct FilterSpec {
   // Variant type for possible filter values.
-  using Value = std::variant<nullptr_t, int64_t, double, const char*>;
+  using Value = std::variant<std::nullptr_t, int64_t, double, const char*>;
 
   // Index of the column in the dataframe to filter.
   uint32_t column_index;
